src/schema/v2/bulkUpdatePartnerArtworksMutation.ts

```diff
@@ -124,7 +113,19 @@ export const updatePartnerArtworksMutation = mutationWithClientMutationId<
     }
 
     try {
-      return await updatePartnerArtworksLoader(id, gravityOptions)
+      const gravityResponse = await updatePartnerArtworksLoader(
+        id,
+        gravityOptions
+      )
+
+      // In the future it could be helpful to have a list of successfully opted in ids, can add this to gravity at a later date
+
+      const formattedReturn = {
+        updatedPartnerArtworks: { count: gravityResponse.success, ids: [] },
```

Comment:
The array of IDs are actually those of artworks that are _not_ updated. If the artwork was successfully updated, then that is what is what was expected and those that weren't updated are the anomaly, hence their IDs as we may be able to use them in a call to action in the future.

Some more context for the endpoint can be found in [this comment](https://github.com/artsy/metaphysics/pull/4269#discussion_r935331223) and in the [gravity code](https://github.com/artsy/gravity/blob/39caa1fca2398a0468273f7aa67230b47d1f0b17/app/services/partner_artwork_service.rb#L32-L47).

Because of that, I would expect this to be coupled with the `gravityResponse.errors` in `skippedPartnerArtworks`

Reply:
I misread this. I thought there was mapping from the `errors.ids` to here, but it is just an empty _ids_ array to make sure that the `updatedPartnerArtworks` and `skippedPartnerArtworks` follow the same format!